app/src/main/java/io/stormbird/wallet/service/TokensService.java

```diff
@@ -133,4 +142,28 @@ public void setCurrentNetwork(int currentNetwork)
     {
         this.currentNetwork = currentNetwork;
     }
+
+    public List<Token> getAllClass(Class<?> tokenClass)
```

Comment:
Naming

Reply:
it gets all tokens of a specific class. Check the return and input types - it takes a class spec and returns a filtered list of Tokens. 

Maybe 'filterByClass' is a better name although inconsistent with the other methods.